libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryWaterfallTests.java

```diff
@@ -0,0 +1,397 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
```

Comment:
Have you checked this?

Reply:
Yes, haven't pushed the changes yet. I don't want it to mark things that I didn't fix yet as "obsolete"